docs/about/contributing/modules.md

```diff
@@ -0,0 +1,395 @@
+# Contributing modules
+
+Learn how to create and contribute Terraform modules to the Coder Registry. Modules provide reusable components that extend Coder workspaces with IDEs, development tools, login tools, and other features.
+
+## What are Coder modules
+
+Coder modules are Terraform modules that integrate with Coder workspaces to provide specific functionality. They are published to the Coder Registry at [registry.coder.com](https://registry.coder.com) and can be consumed in any Coder template using standard Terraform module syntax.
+
+Examples of modules include:
+
+- **IDE integrations**: [`code-server`](https://registry.coder.com/modules/coder/code-server), [`jetbrains-gateway`](https://registry.coder.com/modules/coder/jetbrains-gateway), [`cursor`](https://registry.coder.com/modules/coder/cursor)
+- **Development tools**: [`git-clone`](https://registry.coder.com/modules/coder/git-clone), [`dotfiles`](https://registry.coder.com/modules/coder/dotfiles), [`devcontainers-cli`](https://registry.coder.com/modules/coder/devcontainers-cli)
```

Comment:
I think dotfiles and git-clone should go into the `Workspace utilities` category.